artifacts/centos/centos.go

```diff
@@ -64,6 +64,8 @@ func (c *centos) Inspect() (*api.ArtifactDetails, error) {
 			SHA256Sum:            checksum,
 			DownloadURL:          baseURL + candidate,
 			AdditionalUniqueTags: getAdditionalTags(c.Version, c.Variant, candidate),
+			ImageArchitecture:    "amd64",
+			ImageOS:              "linux",
```

Comment:
Do we need to make `ImageOS` configurable? IIUC we should be okay with making it a constant in `pkg/build/build.go`.

Reply:
Okay.